modular_skyrat/modules/customization/modules/client/preferences.dm

```diff
@@ -2401,6 +2416,32 @@ GLOBAL_LIST_INIT(food, list(
 					ShowLangMenu(user)
 					return TRUE
 
+				if("sverb") //TODO: Get this working
+					var/new_verb = input(user, "Choose your character's speech verb, such as says, hisses, chirps, etc. Do not abuse this feature -- you are expected to stay within the server's HRP guidelines.", "Character Preference")  as text|null
+					if(new_verb)
+						new_verb = reject_bad_name(new_verb) //Stealing this for my own function because fuck that I'm lazy
+						if(!new_verb)
+							to_chat(user, "<font color='red'>Invalid verb. Your verb may only contain the characters A-Z, a-z, - and '.</font>")
+						else if(length(new_verb) < 10 && length(new_verb) > 2)
```

Comment:
three seems very short. Struggling to come up with an example of a three letter verb

Reply:
3-9 characters